### PR TITLE
Update plugin docs with helm chart examples

### PR DESCRIPTION
**Pull request by kbence**

### CODE REVIEW

File: docs/pages/enterprise/workflow/ssh-approval-mattermost.mdx [hunk 1]
```diff
@@ -103,6 +103,15 @@ Run `./install` from `teleport-mattermost` or place the executable in the approp
 </TabItem>
 </Tabs>
 
+<Details title="Installing via Helm?">
+  ```code
```
Comment (gozer) [inline docs/pages/enterprise/workflow/ssh-approval-mattermost.mdx:107]: \```console

File: docs/pages/enterprise/workflow/ssh-approval-slack.mdx [hunk 2]
```diff
@@ -75,6 +75,15 @@ and will require access to both the public internet and the Teleport Auth Servic
 </TabItem>
 </Tabs>
 
+<Details title="Installing via Helm?">
+  ```code
```
Comment (gozer) [inline docs/pages/enterprise/workflow/ssh-approval-slack.mdx:79]: \```console

File: docs/pages/setup/guides/fluentd.mdx [hunk 3]
```diff
@@ -44,11 +44,22 @@ from Teleport's events API, and forwards them to Fluentd.
...
   </TabItem>
+
   <TabItem label="Docker">
   ```code
   $ docker pull quay.io/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
   ```
   </TabItem>
+
+  <TabItem label="Helm">
+  ```code
```
Comment (gozer) [inline docs/pages/setup/guides/fluentd.mdx:55]: \```console

File: docs/pages/setup/guides/fluentd.mdx [hunk 4]
```diff
@@ -226,6 +237,27 @@ $ tctl auth sign --user=teleport-event-handler --out=identity
...
 
+</TabItem>
+<TabItem label="Helm chart" scope={["cloud"]}>
+
+If you are planning to use the Helm chart, you'll need to generate the keys
+with the `file` format, then create a secret in Kubernetes:
+
+Create the identity using the following command:
+
+```code
```
Comment (gozer) [inline docs/pages/setup/guides/fluentd.mdx:248]: \```console

File: docs/pages/setup/guides/fluentd.mdx [hunk 5]
```diff
@@ -226,6 +237,27 @@ $ tctl auth sign --user=teleport-event-handler --out=identity
...
+$ tctl auth sign --format=file --user=teleport-event-handler --out=identity
+```
+
+Then create the Kubernetes secret:
+
+```code
```
Comment (gozer) [inline docs/pages/setup/guides/fluentd.mdx:254]: \```console

**Review by gozer — APPROVED** [review]
Looking great apart from my usual OCD yaml \```console nits.

Sure feels like some of this could/should be templated/generated one day, lots of repeated content.

File: docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx [hunk 6]
```diff
@@ -123,6 +123,15 @@ Run `./install` from `teleport-jira` or place the executable in the appropriate
 </TabItem>
 </Tabs>
 
+<Details title="Installing via Helm?">
+  ```code
```
[the remark this answers is not in the document]
Comment (kbence) [reply docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx:127]: I used `code` to be consistent with the other examples in the files. Based on the current MDX config (https://github.com/gravitational/docs/blob/main/server/mdx-config-docs.ts#L91), the `remark-code-snippet` plugin is enabled for `code` and `bash` only, I could use `bash` of course, just wanted to be consistent, as I mentioned.

[the remark this answers is not in the document]
Comment (kbence) [reply docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx:127]: This plugin allows the snippet to be copied without comments and the prepended `$` symbols, which makes it really useful for pasting it directly to a terminal (if I'm not mistaken). I tried with and without it, and I found this difference.

**Comment by kbence** [comment]
> Sure feels like some of this could/should be templated/generated one day, lots of repeated content.

Yeah, I was thinking the same... There's a feature to include files from the repository, which we use for the config examples mostly, but I'm not sure how we could pass arguments to that.

**Pull request closed without merging**